tests/datasets/test_errors_processor.py

```diff
@@ -251,11 +251,6 @@ def test_error_processor() -> None:
             "this_is_me",
             "snuba",
         ],
-        "_tags_flattened": (
```

Comment:
Shouldn't we still be writing an empty string here? (at least temporarily)

Reply:
In theory no, `input_format_defaults_for_omitted_fields` should take care of that on clickhouse, though we are not 100% sure it works as it should, this is why we are testing this on errors first.

Reply:
i'm confused why we need to rely on an incorrect behavior of clickhouse for this to work? is it because we want to drop the column as soon as this gets merged and there's not another way to achieve that?

Reply:
Actually this was meant to test whether clickhouse is behaving incorrectly or not. But in the meantime I think I found another way to verify that. Hold on, please.

Reply:
No, tested also by passing the settings and forcing it to 0 upon insert 
```
2020-06-22 17:15:54,494 http://localhost:8123 "POST /?load_balancing=in_order&insert_distributed_sync=1&input_format_defaults_for_omitted_fields=0&query=INSERT+INTO+default
.sentry_local+FORMAT+JSONEachRow HTTP/1.1" 200 None
```

And that worked fine, so we should still understand whether this works or not (on the errors dataset that is not used).

Reply:
This PR is actually adding the empty string in case this test does not work
https://github.com/getsentry/snuba/pull/1033/

Reply:
Ok